recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -26,7 +26,7 @@
             :attrs="attrs"
             @click="emojiClicked(reaction)"
           >
-            <span class="d-mr4">
+            <span class="d-mr4 d-d-inline-flex">
```

Comment:
this one is fine to leave since this file is already full of utility classes and we'll have to do a whole migration anyways.

Reply:
Yes they are a lot on our recipes, do we have a ticket to make this migration?

Reply:
Yep it's here https://dialpad.atlassian.net/browse/DLT-1125